src/StripesContext.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+const Context = React.createContext();
+
+export const StripesProvider = Context.Provider;
+export const StripesConsumer = Context.Consumer;
```

Comment:
All things being equal, I think we can just do this in a single line:

```js
export default const Context = React.createContext();
```

Then in the wrapper:

```js
<Context.Consumer>
  { stripes => .....}
</Context.Consumer>
```

And in the provider:

```js
import StripesContext from 'stripes-context';

render() {
  return <StripesContext.Provider> ..... </StripesContext.Provider>
}
```

This is also the recommend in the React docs.